src/main/java/de/embl/cba/mobie/ui/viewer/SourcesDisplayUI.java

```diff
@@ -89,7 +89,7 @@ public static JButton createBrightnessButton( int[] buttonDimensions,
 	}
 
 
-	private static ArrayList< ConverterSetup > getConverterSetups(
+	public static ArrayList< ConverterSetup > getConverterSetups(
```

Comment:
Since it is `public static` now maybe move to some `Utils` class?

Reply:
I moved this to BdvUtils in imagej-utils

Reply:
Cool!